README.md

```diff
@@ -1,3 +1,101 @@
+```
+5. Run the `docker` command to verify the docker instance is running for ElasticSearch, Kibana and ElasticMQ as below:
+```
+docker ps -a
+```
+6. Clone the ElasticMQ Git repository to a suitable location:  
+```
+git clone https://github.com/softwaremill/elasticmq.git
+```
+7. Switch to the `ui` directory in the ElasticMQ project:
+```
+cd ./elasticmq/ui
+```
+8. Install the Yarn dependencies:
+```
+yarn
+```
+9. Start the ElasticMQ UI:
+```
+yarn start
+```
+10. Go to http://localhost:3000/ in a web browser and you should see the message queues configured in `custom.conf`, e.g. `search-queue`, `search-reply-queue`, `search-error-queue`
+
+11. To run the Pan-Archival Catalogue Services locally, it is recommended to first package and stage the application (see [Packaging](#packaging) below) and then navigate to the `stage` directory and follow the instructions given in the [Running](#running) section below. Note that the queue names configured in the `application.conf` file used by application must match those in the `custom.conf` file used by ElasticMQ.
+
+14. Your message service is now running and ready to respond to requests.
+
+## Packaging
+This project makes use of the [sbt-native-packager](https://www.scala-sbt.org/sbt-native-packager/index.html) plugin to create distribution packages of the application.
+
+In order to create a zip package, which includes the application bundled as a jar file, together with its dependent jar files and also scripts to start the application, run the following sbt command:
+```
+sbt universal:packageBin
+```
+The zip file will be created in the `./target/universal` directory of the project.
+It is also possible to preview the distribution using by running:
+```
+sbt stage
+```
+This creates the package files in the `./target/universal/stage` directory from where they can be examined and tested.
+
+It is also possible to build a local Docker image of the application with the following command:
+```
+sbt docker:publishLocal
+```
+You can verify the existence of the Docker image by running the following from the command line:
+```  
+docker image ls
+```
+It is recommended to [read more about the Docker Plugin](https://www.scala-sbt.org/sbt-native-packager/formats/docker.html) if you wish yo use this feature. 
+## Running
+When the application has been packaged as described above, the following directory structure will be created in the distribution:
+```
+│
+└─bin
+│   ctd-omega-services
+│   ctd-omega-services.bat
+└─lib
+uk.gov.nationalarchives.ctd-omega-service.jar
+...
+```
+The `bin` folder contains the scripts to start the application from the command line and the `lib` folder contains the prototype jar file and other dependencies.
+
+Running the application from the command line using the `ctd-omega-services` script or `ctd-omega-service.bat` Windows batch file with the `--help` flag will display further instructions for running the application including the required command line arguments.
```

Comment:
Perhaps add a 'dev' section explaining how to use `sbt` to both run the application (including echo service), and run the tests.

Reply:
I have updated the wording in the README. Hopefully the intention is clearer.

Reply:
Confirmed.